js/keyword-search.js

```diff
@@ -138,18 +143,20 @@ baltimoreCounty.keywordSearch = (function keywordSearch($, sessionStorage, Handl
 		var keyCode = event.which || event.keyCode;
 		var $target = $(event.currentTarget);
 		var $searchResults = $('#header-search-results');
+		var $visibleSearchResults = $searchResults.find('li').is(':visible');
```

Comment:
Change to make it seem like a boolean, rather than a jquery selector ```areSearchResultsVisible```

Reply:
Done